src/vmm/src/builder.rs

```diff
@@ -169,6 +170,29 @@ impl VmmBuilder {
             .map_err(VmmActionError::from)
     }
 
+    /// Updates the path of the host file backing the emulated block device with id `drive_id`.
+    pub fn update_block_device_path(&mut self, drive_id: String, path_on_host: String) -> UserResult {
```

Comment:
2 public functions with same name... also not clear what you do with this function. Isn't this a leftover?

Reply:
I figured it out what you have done here and is fine.